startup/00-base.py

```diff
@@ -163,6 +160,11 @@ def reload(self):
 
 
 # runengine_metadata_dir = appdirs.user_data_dir(appname="bluesky") / Path("runengine-metadata")
-runengine_metadata_dir = Path('/nsls2/xf05id1/shared/config/runengine-metadata')
+runengine_metadata_dir = Path('/nsls2/xf05id1/shared/config/runengine-metadata-new')
```

Comment:
New was added because this was messing with testing going back and forth between the old metadata standard and new metadata standard. Since we have been running on new metadata for a while, the "old" directory can removed